MySQL/system/ksp-audit-mysql-v-235149-mysql-network.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-mysql-v-235149-mysql-network
+  namespace: default    #change default to your namespace
+spec:
+  tags: ["MySQL", "STIG", "V-235149"]
+  message: "MySQL was accessed from network"
+  selector:
+    matchLabels:
+      app: mysql        #change app: mysql to your label
+  network:
+    matchProtocols:
+    - protocol: TCP
+    severity: 6
+    action: Audit
```

Comment:
If we start auditing TCP connections then we will have too many alerts. This policy does not make sense.